[IMP] fiscal_localizations/denmark: Add documentation page for Denmark (#12185)
odoo/documentation: content/applications/finance/fiscal_localizations/denmark.rst

  for the required five-year period, regardless of customer relationship status, bankruptcy, or dissolution.

- **Provide Decryption Capabilities:** Ensure that encrypted bookkeeping data and receipts can be decrypted
  into a structured and readable format.

Odoo Compliance Measures
========================

For companies using Odoo on the Odoo Cloud hosting specifically, the system is designed to meet these requirements
through the following features and processes:

- **Immutable Transaction Records:**
    - Once transactions are recorded, they cannot be deleted through the user interface.
    - All modifications are logged, providing a complete audit trail.
    - While historic dated entries can be made, the system records the creation date and time of the entry.

- **Secure Document Storage:**
    - Receipts and digital vouchers are stored as attachments and integrated into the database, ensuring they
      are included in backups.
    - Posted documents cannot be deleted.
    - We fully support the storage of mandatory digital vouchers as defined by Danish regulations.

- **Continuous Data Availability:**

>     - During active system usage, authorized personnel can access all transactions and digital vouchers through
>       the user interface.
>     - Regardless of customer relations, bankruptcy, or dissolution, Odoo can provide access to transaction and
>       digital voucher details, during 6 years (see more below)

Reviewer: ```suggestion
- Clients with active subscriptions can access all transactions and digital vouchers through Odoo.
- Regardless of customer relations, bankruptcy, or dissolution, Odoo can provide access to transaction and digital voucher details to former clients for six years (see :ref:`localizations/denmark/data-lifecycle`).
```
Several things here:
1. The first bullet was confusing. Let me know if you disagree with this suggestion and we'll find a solution.
2. The second bullet is a little confusing - are we providing a guarantee of access to their data in the case of the client's bankruptcy/dissolution or in the case of Odoo's bankruptcy/dissolution?
3. "...details, during 6 years" - bit of a françaicism ;) "for six years" is better - no need for the comma, and spell out "six".
4. Let's add a reference to make "see more below" more specific.

- **Automated Data Export and Secure Storage:**
    - Odoo Accounting implements no automatic deletion or archival of recorded transactions, so if a company has

>       been recording transactions for 6 years, the 6 years of history are preserved in the Odoo Accounting database.

Reviewer: ```suggestion
      been recording transactions for six years, the six years of history are preserved in the Odoo Accounting database.
```
Let's spell out numbers ten and under. There are several other cases of this - I'll let you find them so that I don't clutter the PR with even more comments :)

Author: I changed all the ones that are not relative to a specific article / §

    - As described in the `Odoo SLA <https://www.odoo.com/cloud-sla>`_ and
      `Odoo Privacy Policy <https://www.odoo.com/privacy>`_, the Odoo Cloud relies on immutable daily snapshot
      backups, which cannot be individually altered or deleted, even at the customer's request, ensuring their integrity.
    - All documents and receipts stored in a database backup are available as a standard ZIP archive, accompanying
      the SQL dump.

> 

Reviewer: ```suggestion

.. _localizations/denmark/data-lifecycle:

```
Adding the custom anchor so we can refer to it above

- **Data Lifecycle Management:**
    - Odoo database backups are available in standard SQL dump formats at all times, and include all recorded
      transactions.
    - The default Odoo Cloud SLA guarantees 3 months of backup history to all active customers. For the avoidance
      of doubt, each such backup contains the complete bookkeeping history of the company. As a special guarantee for

>       Denmark companies subject to the DBA and who opt for an Odoo Cloud solution, the backup retention gets increased

Reviewer: ```suggestion
      Danish companies subject to the DBA and who opt for an Odoo Cloud solution, the backup retention gets increased
```

  and digital voucher details to former clients for six years (see :ref:`localizations/denmark/data-lifecycle`).

Automated Data Export and Secure Storage
----------------------------------------

- Odoo Accounting implements no automatic deletion or archival of recorded transactions, so if a company has
  been recording transactions for six years, the six years of history are preserved in the Odoo Accounting database.
- As described in the `Odoo SLA <https://www.odoo.com/cloud-sla>`_ and
  `Odoo Privacy Policy <https://www.odoo.com/privacy>`_, the Odoo Cloud relies on immutable daily snapshot
  backups, which cannot be individually altered or deleted, even at the customer's request, ensuring their integrity.
- All documents and receipts stored in a database backup are available as a standard ZIP archive, accompanying
  the SQL dump.

.. _localizations/denmark/data-lifecycle:

Data Lifecycle Management
-------------------------

- Odoo database backups are available in standard SQL dump formats at all times, and include all recorded
  transactions.
- The default Odoo Cloud SLA guarantees three months of backup history to all active customers. For the avoidance

>   of doubt, each such backup contains the complete bookkeeping history of the company. As a special guarantee for

Reviewer 2: I'm not sure what you mean by "for the avoidance of doubt", would we clarify this please? Do you mean:
- To ensure complete data integrity, or
- To enable complete and accurate data recovery

(I'd pick the most accurate one and use it instead)

Author: Hey, I'm not sure too as I only did the PR for @Olivier-olma 
So I'll ask him!

Reviewer 3: The idea of "avoidance of doubt" is more to clarify what we have in the backup kept. Because Danish government want to be sure that user knows what's in the backup (and required by law)

Reviewer 2: Can we drop it entirely then? Since the rest of the sentence is pretty self-explanatory and says exactly what you explained?

Author: @Olivier-olma :point_up: small up

>   Danish companies subject to the DBA and who opt for an Odoo Cloud solution, the backup retention gets increased

Reviewer 2: ```suggestion
  Danish companies subject to the :abbr:`DBA (Danish Bookkeeping Act)` and who opt for an Odoo Cloud solution, the backup retention gets increased
```
not 100% necessary but it doesn't hurt :wink:

  to six years as soon as they decide to terminate their Odoo Cloud subscription, in order to comply with the
  requirements of Annex 1, 4 of Executive Order 97.
- Companies who are using Odoo products outside of the Odoo Cloud are responsible for implementing their

>   own compliance with the DBA.

Reviewer 2: ```suggestion
  own compliance with the :abbr:`DBA (Danish Bookkeeping Act)`.
```
same as above

Decryption
----------

- Odoo Accounting customer data on the Odoo Cloud is always stored in encrypted form (encryption at rest at

>   storage level). When backups are retrieved, they are automatically decrypted and provided in decrypted form in

Reviewer 2: I'd remove the bullet here since it's the only item in the list
